src/datasources/system/SystemDataSource.ts

```diff
@@ -7,22 +7,25 @@ import {
   FieldType,
   CoreApp,
   toDataFrame,
+  MetricFindValue,
 } from '@grafana/data';
 
-import { TestingStatus, getBackendSrv } from '@grafana/runtime';
+import { TestingStatus, getBackendSrv, getTemplateSrv } from '@grafana/runtime';
 
-import { QueryType, SystemMetadata, SystemQuery, SystemSummary } from './types';
+import { AuthResponse, QueryType, SystemMetadata, SystemQuery, SystemSummary, VariableQuery, Workspace } from './types';
 import { defaultProjection } from './constants';
 import { NetworkUtils } from './network-utils';
 
 export class SystemDataSource extends DataSourceApi<SystemQuery> {
   baseUrl: string;
+  aliasUrl: string;
   constructor(private instanceSettings: DataSourceInstanceSettings) {
     super(instanceSettings);
     this.baseUrl = this.instanceSettings.url + '/nisysmgmt/v1';
+    this.aliasUrl = this.instanceSettings.url + '/niauth/v1';
```

Comment:
```suggestion
    this.authUrl = this.instanceSettings.url + '/niauth/v1/auth';
```
'Alias' doesn't really make sense here. Also, the `/auth` route is likely the only one we'll use from this service, so we can just fully qualify it here instead of constructing it when we make the request.